packages/cozy-konnector-libs/src/libs/saveBills.spec.js

```diff
@@ -76,3 +77,120 @@ describe('saveBills', function() {
     ])
   })
 })
+
+describe('appendContractsDatas', function() {
+  it('Append contracts datas if contractId present in options', async () => {
+    let entries = [
+      {
+        filename: 'filename',
+        fileurl: 'fileurl',
+        amount: 12,
+        date: new Date(),
+        vendor: 'vendor'
+      }
+    ]
+    let options = { contractId: 'contractId' }
+    entries = appendContractsDatas(entries, options)
+    expect(entries[0].contractId).toEqual('contractId')
+    expect(entries[0].contractLabel).toEqual('contractId')
+    expect(options.subPath).toEqual('contractId')
+  }),
+    it('Append contracts datas if contractId & contractLabel presents in options', async () => {
```

Comment:
Why is there a comma just before the `it` ?

Reply:
Thought it was mandatory, test say no, i removing it.